sdk/azidentity/azure_cli_credential.go

```diff
@@ -163,26 +163,21 @@ var defaultAzTokenProvider azTokenProvider = func(ctx context.Context, scopes []
 
 func (c *AzureCLICredential) createAccessToken(tk []byte) (azcore.AccessToken, error) {
 	t := struct {
-		AccessToken      string `json:"accessToken"`
-		Authority        string `json:"_authority"`
-		ClientID         string `json:"_clientId"`
-		ExpiresOn        string `json:"expiresOn"`
-		IdentityProvider string `json:"identityProvider"`
-		IsMRRT           bool   `json:"isMRRT"`
-		RefreshToken     string `json:"refreshToken"`
-		Resource         string `json:"resource"`
-		TokenType        string `json:"tokenType"`
-		UserID           string `json:"userId"`
+		AccessToken string `json:"accessToken"`
+		Expires_On  int    `json:"expires_on"`
```

Comment:
Should this be `int64`? On 64-bit systems it will be 64 bits wide, but on a 32-bit system it will be 32 bits wide. Will the value always fit into a 32-bit int?

Reply:
It will until 2038 i.e. the Epochalypse, because the value is a Unix timestamp which won't overflow 32 bits before then. But sure, this may as well be `int64`.